installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java

```diff
@@ -66,13 +73,21 @@ private long getInstallationSize()
          Pack p = (Pack) iter.next();
          totalSize += p.nbytes;
       }
-     
+      
+      // Account for any additional features in the total aggregate size.
+      String addFeatureSizes = idata.getVariable("INSTALL_ADDITIONAL_SIZES");
```

Comment:
If you want execute tests there should be launch item available already in toolbar:
![image](https://cloud.githubusercontent.com/assets/620330/18803844/e63df51a-81a8-11e6-9ed7-8c754e889421.png)
or in Run Configuration Dialog:
![image](https://cloud.githubusercontent.com/assets/620330/18803860/204d92d8-81a9-11e6-874e-35ddf54a3b4b.png)